godtools/App/Features/Dashboard/Presentation/Dashboard/DashboardTabBarView/DashboardTabBarView.swift

```diff
@@ -16,23 +16,20 @@ struct DashboardTabBarView: View {
         
         ZStack {
             HStack {
-                
-                doubleSpacer()
-                
+                                
                 DashboardTabItem(tabType: .lessons, title: viewModel.lessonsTabTitle, imageName: ImageCatalog.toolsMenuLessons.name, selectedTab: $viewModel.selectedTab)
-                
-                tripleSpacer()
+                    .frame(maxWidth: .infinity)
                 
                 DashboardTabItem(tabType: .favorites, title: viewModel.favoritesTabTitle, imageName: ImageCatalog.toolsMenuFavorites.name, selectedTab: $viewModel.selectedTab)
-                
-                tripleSpacer()
-                
+                    .frame(maxWidth: .infinity)
+                                
                 DashboardTabItem(tabType: .allTools, title: viewModel.allToolsTabTitle, imageName: ImageCatalog.toolsMenuAllTools.name, selectedTab: $viewModel.selectedTab)
+                    .frame(maxWidth: .infinity)
                 
-                doubleSpacer()
             }
             .padding(.top, 16)
             .padding(.bottom, 8)
+            .padding([.leading, .trailing], 8)
             .frame(maxWidth: .infinity)
             .background(
                 Color.white
```

Comment:
I noticed the shadow appears to have a white backing behind it which looks odd when content is clipped behind the shadow.  I changed the HStack to a red background and it's more evident.

![iphone8-ios16](https://user-images.githubusercontent.com/59846460/215108152-fcdafa1e-1542-4f63-b45d-b623e37aaa6f.png)


![iphone8-ios16-red](https://user-images.githubusercontent.com/59846460/215108167-dc73d34c-cced-4fc5-82dc-0adbf00c0aad.png)

Reply:
Good catch, fixed!